includes/admin/class-wc-admin-notices.php

```diff
@@ -404,7 +404,7 @@ public static function wootenberg_feature_plugin_notice() {
 	 * @since  3.5.1
 	 */
 	protected static function is_ssl() {
-		$shop_page = 0 < wc_get_page_id( 'shop' ) ? get_permalink( wc_get_page_id( 'shop' ) ) : get_home_url();
+		$shop_page = wc_get_page_permalink( 'shop', get_home_url() );
```

Comment:
`get_home_url()` is the default fallback URL for `wc_get_page_permalink` so

`wc_get_page_permalink( 'shop', get_home_url() )` and `wc_get_page_permalink( 'shop' )` produce the same URL. It could be removed from the 4 changes.

Reply:
Yea I left it intentionally for easier understanding of the code, anyway I removed it now.